src/pykx/pandas_api/pandas_meta.py

```diff
@@ -154,6 +155,29 @@ def mean(self, axis: int = 0, numeric_only: bool = False):
             tab
         )
 
+    @api_return
+    def std(self, axis: int = 0, ddof: int = 1, numeric_only: bool = False):
```

Comment:
I couldn't find a test for this function at `test_pandas_api.py`, or at least its name was not `test_pandas_std` (as expected). @nipsn can you take a look at it?

Reply:
It's called `test_std`, as expected from the functions in the same area that perform similar computations: `test_mean`, `test_avg`, `test_median`, etc.